### PR TITLE
Add GitLab Writing Day session for WTD PDX 2020

### PR DESCRIPTION
**Pull request by mjang**

### CODE REVIEW

**Comment by plaindocs** [comment]
Looks like you maybe need to bump the heading up one level?

image

I see it as

* You project here
* TGD
* Gitlab 

should all be at the same level.

**Comment by mjang** [comment]
@plaindocs thanks for the catch. I think I've updated correctly. Before merging, I'm trying to see if I can get someone else from GitLab to review this. 

(FWIW, I'm having trouble building locally because I'm having trouble with ablog)

**Comment by plaindocs** [comment]
Hi @mjang 

You can use the RTD preview here https://writethedocs-www--1283.org.readthedocs.build/

Or drop me a line and I'll see if I can help with sphinx/ablog? Have you done a pip install lately? There was a major version sphinx update a month or so ago that might need it.

**Comment by plaindocs** [comment]
I've just fixed the merge conflict also.

File: docs/conf/portland/2020/writing-day.rst [hunk 1]
```diff
@@ -24,9 +24,29 @@ You've just been asked by your team to create documentation for your project. Bu
...
+
+`GitLab <https://about.gitlab.com>`_ is an open core single application for
+the entire software development lifecycle. From project planning and source code
+management to CI/CD, monitoring, and security.
+
+Our `documentation <https://docs.gitlab.com>`_ is open source and everybody can
+contribute to it. All the content is hosted on GitLab.com, so you will need a
+`GitLab.com account <https://gitlab.com/users/sign_up>`_.
+
+For more information, see our `GitLab issue <https://gitlab.com/gitlab-org/technical-writing/-/issues/217>`.
```
Comment (mjang) [inline docs/conf/portland/2020/writing-day.rst]:
```suggestion
For more information, see our `GitLab issue <https://gitlab.com/gitlab-org/technical-writing/-/issues/217>`_.
```

**Pull request merged**